src/main/features/claim/routes/pay-by-account.ts

```diff
@@ -106,6 +120,21 @@ export default express.Router()
           Cookie.saveCookie(req.signedCookies.legalRepresentativeDetails, res.locals.user.id, legalRepDetails),
           CookieProperties.getCookieParameters())
 
-        await saveClaimHandler(res, next)
+        const payClient: PayClient = await getPayClient()
+
+        const paymentResponse: PaymentResponse = await payClient.create(
+          res.locals.user,
+          draft.document.feeAccount.reference,
+          draft.document.externalId,
+          draft.document.yourReference.reference,
+          feeResponse
+        )
+
+        if (paymentResponse.isSuccess) {
```

Comment:
jira seems to say this should be done after saving the case

Reply:
Jira needs to be updated.